saya/core/src/prover/atlantic/mod.rs

```diff
@@ -27,3 +29,17 @@ impl AtlanticProof for String {
         Ok(raw_proof)
     }
 }
+
+/// Calculate the job size based on the number of steps in the pie.
+/// Check the [Atlantic Prover](https://docs.herodotus.cloud/atlantic/sending-query) documentation for more information.
+/// We can use bigger sizes for small pies as well, but this increases the cost.
+/// The sizes have impact on amount of recoursces dedicated to the job.
+/// The sizes are XS, S, M, L.
+pub fn calculate_job_size(pie: CairoPie) -> AtlanticJobSize {
+    match pie.execution_resources.n_steps {
```

Comment:
Let's add in the comment that `XS` is purely virtual for Atlantic optimization, but `XS` and `S` have he exact same cost in SHARP.